website/src/pages/index.js

```diff
@@ -1,104 +1,79 @@
 import React from 'react';
-import clsx from 'clsx';
 import Layout from '@theme/Layout';
 import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import styles from '../css/styles.module.css';
 
-const features = [
-  {
-    title: 'Easy to Use',
-    description: (
-      <div>
-        <p>Describe your software project just once, using Premake's simple and easy to read syntax, and build it everywhere.</p>
-        <p>Generate project files for Visual Studio, GNU Make, Xcode, Code::Blocks, and more across Windows, Mac OS X, and
-        Linux. Use the full featured Lua scripting engine to make build configuration tasks a breeze.</p>
-      </div>
-    ),
-  },
-  {
-    title: 'Version 5.0 (alpha)',
-    description: (
-      <div>
-        <p>The latest iteration of Premake, featuring an improved platform system, support for third-party modules, and loads of new features.</p>
-		    <p className={styles.bigText}>
-          <a href='download/'>Download</a>&nbsp;·&nbsp;<a href='docs/'>Documentation</a>
-        </p>
-      </div>
-    ),
-  },
-  {
-    title: 'Version 4.4 (beta)',
-    description: (
-      <div>
-        <p>The latest in Premake's 4.x stable branch.</p>
-		    <p className={styles.bigText}>
-          <a href='download/'>Download</a>&nbsp;·&nbsp;<a href='https://github.com/premake/premake-4.x/wiki'>Documentation</a>
-        </p>
-      </div>
-    ),
-  },
-];
+import { Container, Column, Row } from '../components/Grid';
+
+
+const Banner = () =>
+	<Container className="banner">
+		<img className="feature-image" src="/img/premake-logo.png" alt="Premake logo" />
+		<h1 className="hero__title">Premake</h1>
+		<p className="hero__subtitle">Powerfully simple build configuration</p>
+		<div className=".buttons">
+			<Link
+				className="button button--outline button--primary button--lg"
+				to="docs/">
+				Get Started
+			</Link>
+			&nbsp;&nbsp;
+			<Link
+				className="button button--outline button--primary button--lg"
+				to="download">
+				Download
+			</Link>
+		</div>
+	</Container>;
+
+
+const Feature = ({ title, children }) =>
+	<div className="col col--4">
+		<h3>{title}</h3>
+		<div>{children}</div>
+	</div>;
```

Comment:
I suggest using `<Column> </Column>` here. It's already imported but it's not used (yet).